content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -584,6 +584,12 @@ but this is more complex and error-prone and is not recommended.
 
 === Using third-party libraries
 
+[IMPORTANT]
+====
+While possible, accessing third-party libraries using `@Grab` from trusted libraries has various issues and is not recommended.
```

Comment:
We could go into some of the issues (class loading conflicts, proxies/connectivity issues, performance impact on master, etc.), but I wasn't sure how much detail would be useful here.